python/lsst/ctrl/bps/wms/htcondor/lssthtc.py

```diff
@@ -349,23 +411,19 @@ def htc_version():
     return f"{int(version_info.group(1))}.{int(version_info.group(2))}.{int(version_info.group(3))}"
 
 
-def htc_submit_dag(htc_dag, submit_options=None):
-    """Create DAG submission and submit
+def htc_submit_dag(sub):
+    """Submit job for execution.
 
     Parameters
     ----------
-    htc_dag : `HTCDag`
-        DAG to submit to HTCondor
-    submit_options : `dict`
-        Extra options for condor_submit_dag
-    """
-    ver = version.parse(htc_version())
-    if ver >= version.parse("8.9.3"):
-        sub = htcondor.Submit.from_dag(htc_dag.graph["dag_filename"], submit_options)
-    else:
-        sub = _htc_submit_dag_old(htc_dag.graph["dag_filename"], submit_options)
+    sub : `htcondor.Submit`
+        An object representing a job submit description.
 
-    # Submit DAG to HTCondor's schedd.
+    Returns
+    -------
+    schedd_job_info : `dict` [`str`, `dict` [`str`, `dict` [`str` Any]]]
+        Information about the submitted job.
```

Comment:
Perhaps it's easy to find the information somewhere else, but if not it would be nice if the description very, very briefly described the nested dictionaries  (e.g.,  per schedd -> per dagman job -> job ad key/values )